src/app/shared/services/iam.service.ts

```diff
@@ -187,7 +187,10 @@ export class IamService {
         this.cacheClient = cacheClient;
         this.verifiableCredentialsService = verifiableCredentialsService;
         if (createDocument) {
-          const { didRegistry, claimsService } = await connectToDidRegistry();
+          const { didRegistry, claimsService } = await connectToDidRegistry(
+             // @ts-ignore
+            this.configureIpfsConfig()
```

Comment:
We won't need this ignore once ICL pr is in and version is bumped, but maybe this would work now before it's in as John suggested.